root/static/scripts/edit/externalLinks.js

```diff
@@ -966,13 +1110,33 @@ export class ExternalLink extends React.Component<LinkProps> {
               onTypeChange={props.onTypeChange}
               onVideoChange={props.onVideoChange}
               typeOptions={props.typeOptions}
+              urlMatchesType={props.urlMatchesType}
             />
         ))}
+        {firstLink.pendingTypes &&
+          <tr>
+            <td />
+            <td>
+              {exp.l(
+                `The following type(s) will be automatically selected
+                 when submitting: {types}`,
+                {
```

Comment:
I don't quite understand this message.  It doesn't seem I can override this behavior at all, so I'm not sure why it's telling me this instead of just selecting every type.  The "when submitting" part is also confusing, since "submit" to me is when I submit the form, but these are just selected when I move focus.